server/server/src/scheduled_tasks.rs

```diff
@@ -0,0 +1,32 @@
+use service::service_provider::ServiceProvider;
+use std::{sync::Arc, time::Duration};
+use tokio::task::JoinHandle;
+
+static TASK_INTERVAL: Duration = Duration::from_secs(10);
+
+pub fn spawn_scheduled_task_runner(service_provider: Arc<ServiceProvider>) -> JoinHandle<()> {
+    tokio::spawn(async move {
+        scheduled_task_runner(service_provider).await;
+    })
+}
+
+async fn scheduled_task_runner(service_provider: Arc<ServiceProvider>) {
+    let mut interval = actix_web::rt::time::interval(TASK_INTERVAL);
+    let service_context = service_provider.basic_context().unwrap();
+
+    loop {
+        interval.tick().await;
+        log::debug!("Processing Scheduled Tasks");
+        let send_emails = service_provider
+            .email_service
+            .send_queued_emails(&service_context);
+        match send_emails {
+            Ok(num) => {
+                if num > 0 {
+                    log::info!("Sent {} queued emails", num);
+                }
+            }
+            Err(error) => log::error!("Error sending queued emails: {:?}", error),
```

Comment:
When not configured, we get almost duplicated errors here...
```
2025-01-06 14:29:05.157412000 [ERROR] <service::email:118>:Email settings not configured
2025-01-06 14:29:05.157883000 [ERROR] <server::scheduled_tasks:29>:Error sending queued emails: NotConfigured
```
Maybe remove the other one and keep this?

Reply:
Done [34f4aeb](https://github.com/msupply-foundation/open-msupply/pull/5915/commits/34f4aebee2c542578429f74a9b920480328a5c1b)